src/GraphQL/Schemas/Primary/Mutations/SendTransactionMutation.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Enjin\Platform\GraphQL\Schemas\Primary\Mutations;
+
+use Closure;
+use Enjin\Platform\Enums\Global\TransactionState;
+use Enjin\Platform\Exceptions\PlatformException;
+use Enjin\Platform\GraphQL\Schemas\Primary\Traits\InPrimarySchema;
+use Enjin\Platform\Interfaces\PlatformGraphQlMutation;
+use Enjin\Platform\Models\Transaction;
+use Enjin\Platform\Rules\ValidHex;
+use Enjin\Platform\Services\Blockchain\Implementations\Substrate;
+use Enjin\Platform\Services\Database\TransactionService;
+use Facades\Enjin\Platform\Services\Database\WalletService;
+use GraphQL\Type\Definition\ResolveInfo;
+use GraphQL\Type\Definition\Type;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Str;
+use Rebing\GraphQL\Support\Facades\GraphQL;
+use Rebing\GraphQL\Support\Mutation;
+
+class SendTransactionMutation extends Mutation implements PlatformGraphQlMutation
+{
+    use InPrimarySchema;
+
+    /**
+     * Get the mutation's attributes.
+     */
+    public function attributes(): array
+    {
+        return [
+            'name' => 'SendTransaction',
+            'description' => __('enjin-platform::mutation.update_transaction.description'),
+        ];
+    }
+
+    /**
+     * Get the mutation's return type.
+     */
+    public function type(): Type
+    {
+        return GraphQL::type('String!');
+    }
+
+    /**
+     * Get the mutation's arguments definition.
+     */
+    public function args(): array
+    {
+        return [
+            'id' => [
+                'type' => GraphQL::type('Int'),
+                'defaultValue' => null,
+            ],
+            'signingPayloadJson' => [
+                'type' => GraphQL::type('Object'),
```

Comment:
Was thinking if you can just create a SigningPayloadJsonInput type and define all the fields required so you don't need to do this

```
   if (!Arr::has((array) $payload, ['address', 'method', 'nonce', 'era', 'tip'])) {
        throw new PlatformException(__('enjin-platform::error.signing_payload_json_is_invalid'));
  }
```

Reply:
Also should this always be required?

Reply:
The problem with making an input type is not accepting "useless" fields.... Since this comes from external sources, it may come with additional fields and we don't want to throw a error just because the person has sent extra fields that will not be used.... The other possibility would be to use a JSON field but then we would not get the nice formatting in graphiql 
![image](https://github.com/enjin/platform-core/assets/5619696/927c1b29-dc50-4c5a-aaab-3a4caaf1f917)

Reply:
How about making this field always required?

Reply:
Sure